package.json

```diff
@@ -30,14 +30,13 @@
     "@nestjs/testing": "6.10.13",
     "@types/jest": "24.0.25",
     "@types/node": "13.1.4",
-    "@types/stripe": "7.13.19",
```

Comment:
Added Type Definitions https://github.com/stripe/stripe-node/releases/tag/v8.0.0

Reply:
Can you update this dependency in the example app as well?

Reply:
thank you review, fix it https://github.com/dhaspden/nestjs-stripe/pull/82/commits/e4629d4e33f5b09c40b75fa9265fda763e253f2a :)